org-super-agenda.el

```diff
@@ -238,9 +242,16 @@ If ANY is non-nil, return as soon as FORM returns non-nil."
   "Return list of tags in agenda item string S."
   (org-find-text-property-in-string 'tags s))
 
+(defun org-super-agenda--make-agenda-header-separator ()
+  "Return a string made from `org-super-agenda-header-separator'."
+  (let ((s org-super-agenda-header-separator))
+    (typecase s
+      (character (concat (make-string (window-width) s) "\n"))
+      (string s))))
+
 (defun org-super-agenda--make-agenda-header (s)
   "Return agenda header containing string S.
-Prepended with `org-super-agenda-header-separator'."
+Prepended with string made from `org-super-agenda-header-separator'."
```

Comment:
Let's leave this docstring unchanged, as the user can see the details from the `defcustom` or the source code.

Reply:
I am afraid that this will be misleading since `org-super-agenda-header-separated` is not directly used.

Reply:
Good point.  Let's do it the standard Emacs way, then.  How about:

```el
"Return agenda header containing string S.
Prepended with `org-super-agenda-header-separator', which see."
```

Reply:
Changed.